### DISCUSSION TITLE
Raise Governace Limit For Solana

### DISCUSSION THREAD

**Discussion by mizdler** [wormhole-foundation/wormhole#3761] [Governance]
Since we hit the bridge limit on Solana, I suggest increasing the 24 hour limit to 50m (same as Ethereum)

**Comment by SimonKraus** [comment]
That's just a logical step after Solana is outperforming Ethereum in terms of DeFi volumes. 
Wouldn't be surprised seeing Solana passing the 50M limit before Ethereum in the next step. 

As you all know Staking Facilities is a big believer in Solana DeFi and strongly supports this limit raise.
